integration/.graphqlrc.yml

```diff
@@ -18,7 +18,4 @@ extensions:
     ## Codegen plugins
     plugins:
```

Comment:
This should not be needed but kind of added it here to not ran through this issuehttps://github.com/aerogear/graphback/issues/1296

Reply:
Remove this hack once https://github.com/aerogear/graphback/pull/1569 is merged.